scipy/linalg/_solvers.py

```diff
@@ -150,7 +150,7 @@ def _solve_discrete_lyapunov_bilinear(a, q):
 
 def solve_discrete_lyapunov(a, q, method=None):
     """
-    Solves the discrete Lyapunov equation :math:`A'XA-X=-Q`.
+    Solves the discrete Lyapunov equation :math:`AXA^H - X = -Q`.
```

Comment:
I suggest to write the equation as `AXA^H - X + Q = 0` (looks more pleasing to me) and change the description of `q` in the parameters section in some way (+ mention its Hermitian property).